samples/zigbee/light_bulb/src/main.c

```diff
@@ -185,35 +185,42 @@ ZB_HA_DECLARE_DIMMABLE_LIGHT_CTX(
 	dimmable_light_ctx,
 	dimmable_light_ep);
 
-/**@brief Callback for button events.
+/**@brief Starts identifying the device.
  *
- * @param[in]   button_state  Bitmask containing buttons state.
- * @param[in]   has_changed   Bitmask containing buttons
- *                            that have changed their state.
+ * @param  bufid  Unused parameter, required by ZBOSS scheduler API.
  */
-static void button_changed(uint32_t button_state, uint32_t has_changed)
+static void start_identifying(zb_bufid_t bufid)
 {
 	zb_ret_t zb_err_code;
 
-	/* Calculate bitmask of buttons that are pressed
-	 * and have changed their state.
+	ZVUNUSED(bufid);
+
+	/* Check if endpoint is in identifying mode,
+	 * if not, put desired endpoint in identifying mode.
 	 */
+	if (dev_ctx.identify_attr.identify_time == ZB_ZCL_IDENTIFY_IDENTIFY_TIME_DEFAULT_VALUE) {
+		LOG_INF("Enter identify mode");
+
+		zb_err_code = zb_bdb_finding_binding_target(HA_DIMMABLE_LIGHT_ENDPOINT);
+		ZB_ERROR_CHECK(zb_err_code);
+	} else {
+		LOG_INF("Cancel identify mode");
+		zb_bdb_finding_binding_target_cancel();
+	}
+}
+
+/**@brief Callback for button events.
+ *
+ * @param[in]   button_state  Bitmask containing buttons state.
```

Comment:
`Bitmask containing state of the buttons` if plural, 
or `Bitmask containing the button's` if singular.

Reply:
Changed to your suggestion